### ISSUE TITLE
Does this package need linking?

### ISSUE THREAD

**Issue by iuliuvisovan** [robinpowered/react-native-screen-brightness#29]
I'm confused. Why don't the docs say this anywhere. Very poor documentation.

I tried to link this package and it crashed, tried to do it manually but it fails at the build:

> > Task :react-native-screen-brightness:compileDebugJavaWithJavac
> /Users/iuliuvisovan/Desktop/olly-react/node_modules/react-native-screen-brightness/android/src/main/java/com/robinpowered/react/ScreenBrightness/ScreenBrightnessModule.java:28: error: ScreenBrightnessModule is not abstract and does not override abstract method onNewIntent(Intent) in ActivityEventListener
> public class ScreenBrightnessModule extends ReactContextBaseJavaModule
>        ^
> /Users/iuliuvisovan/Desktop/olly-react/node_modules/react-native-screen-brightness/android/src/main/java/com/robinpowered/react/ScreenBrightness/ScreenBrightnessModule.java:64: error: method does not override or implement a method from a supertype
>     @Override
>     ^
> /Users/iuliuvisovan/Desktop/olly-react/node_modules/react-native-screen-brightness/android/src/main/java/com/robinpowered/react/ScreenBrightness/ScreenBrightnessPackage.java:33: error: method does not override or implement a method from a supertype
>   @Override
>   ^
> 3 errors
> 
> 
> FAILURE: Build failed with an exception.

**Comment by phamhoaivu911** [comment]
hi @ajwhite , could you please help on this error?

**Comment by ajwhite** [comment]
Yes, this project does require linking (and in need of proper documentation for that process)

What version of React Native are you running? Those errors sound like a possible compatibility issue

**Comment by dasoga** [comment]
Same issue, did you find a solution here? 
Thanks!

**Comment by dirkpostma** [comment]
Same issue here.


```
$ react-native info

  React Native Environment Info:
    System:
      OS: macOS 10.14.5
      CPU: (12) x64 Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz
      Memory: 506.47 MB / 16.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 10.14.1 - ~/.nvm/versions/node/v10.14.1/bin/node
      Yarn: 1.15.2 - /usr/local/bin/yarn
      npm: 6.4.1 - ~/.nvm/versions/node/v10.14.1/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.2, macOS 10.14, tvOS 12.2, watchOS 5.2
      Android SDK:
        API Levels: 23, 25, 26, 27, 28
        Build Tools: 19.1.0, 26.0.3, 27.0.3, 28.0.3
        System Images: android-19 | Google APIs Intel x86 Atom, android-23 | Google APIs Intel x86 Atom_64, android-28 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: 3.2 AI-181.5540.7.32.5056338
      Xcode: 10.2.1/10E1001 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.6.3 => 16.6.3
      react-native: 0.57.8 => 0.57.8
    npmGlobalPackages:
      react-native-cli: 2.0.1
      react-native-git-upgrade: 0.2.7
```
